index.js

```diff
@@ -0,0 +1,59 @@
+let form = document.getElementById("new-task-form");
+let input = document.getElementById("new-task-input");
+let message = document.getElementById("message");
+let posts = document.getElementById("tasks");
+
+form.addEventListener("submit", (e) => {
+  e.preventDefault();
+  formValidation();
+});
+let formValidation = () => {
+  if (input.value === "") {
+    message.innerHTML = "Input field can not be blank...";
+  } else {
+    message.innerHTML = "";
+    acceptData();
+  }
+};
+let data = [];
+
+let acceptData = () => {
```

Comment:
if this funciton is sotring data , then it sould be named storeData or something similar